saleor/plugins/avatax/plugin.py

```diff
@@ -573,6 +572,17 @@ def calculate_order_line_unit(
             / quantity,
         )
 
+    @property
+    def _tax_included(self) -> Callable[[], bool]:
+        configuration = {item["name"]: item["value"] for item in self.configuration}
```

Comment:
Do you need this? Can't you set this value in `__init__` method? This is something that will be used in all calculations method.

Reply:
I took this approach from the [__init__](https://github.com/saleor/saleor/blob/main/saleor/plugins/avatax/plugin.py#L144). The `self.configuration` already exists and has a different structure.

What do you expect @korycins ? Another attribute like `self._configuration`? I rather prefer a temp structure.

Reply:
@mociepka Personally, I would consider adding new fields to `self.configuration` (can be with `_` also). 
And maybe adding a function to AvataxConfiguration dataclass `tax_included`, that will return you info, if the tax should be included in prices or not - especially since now, Avatax plugin has its own configuration to change this approach. 
Then you don't need to have the parameter `tax_included` in functions/methods.

Reply:
@korycins pushed new changes.